worlds/minecraft/__init__.py

```diff
@@ -84,7 +84,8 @@ class MinecraftWorld(World):
     victory!
     """
     game: str = "Minecraft"
```

Comment:
It's already declared a string on the class
```suggestion
    game = "Minecraft"
```

Reply:
can one have to many type declarations?

Reply:
So true